Files/UserControls/Widgets/DrivesWidget.xaml

```diff
@@ -132,6 +132,7 @@
                                 </Grid.ColumnDefinitions>
                                 <FontIcon
                                     Grid.Column="0"
+                                    x:Phase="2"
```

Comment:
I believe x:Phase makes more sense for big lists that get scrolled, drive list is not big and all the items are always visible at once on screen. Does adding x:Phase make sense here?
[We should really to do it on the GridViewBrowser/GenericFileBrowser template though]

Reply:
The drives list can get a little long especially when there are a lot of network drives 😃. We should work on this for the GridViewBrowser/GenericFileBrowser templates as well.